js/index.js

```diff
@@ -94,10 +94,8 @@ carrouselArrows.forEach(function (element) {
 /* Function about the arrow down or drop down wherever you want
 */
 function goDown() {
-  console.log('click');
-  
-  let lookdown = document.getElementById('carrousel-section');//take the destini to scroll
-  lookdown.scrollIntoView({ block: "start", behavior: "smooth" });
+  let lookdown = document.getElementById('lookdown-scroll');//take the destini to scroll
```

Comment:
Change this name variable to `anchorBtn` and Id to `homepage-hero__anchor`

Reply:
done